cfme/tests/configure/test_timeprofile.py

```diff
@@ -71,8 +77,12 @@ def test_hours_required_error_validation():
         timezone="(GMT-10:00) Hawaii",
         days=True,
         hours=False)
-    with error.expected("At least one Hour must be selected"):
-        tp.create()
+    if version.current_version() > "5.7":
```

Comment:
I think we should be looking for the validation field text and not just that the field is dimmed. If for some reason the locator breaks and we don't fill the form in, the add button will still be dimmed right?